go/kbfs/libkbfs/disk_block_cache.go

```diff
@@ -199,6 +200,7 @@ func newDiskBlockCacheLocalFromStorage(
 	blockDbOptions := *leveldbOptions
 	blockDbOptions.CompactionTableSize = defaultBlockCacheTableSize
 	blockDbOptions.BlockSize = defaultBlockCacheBlockSize
+	blockDbOptions.Filter = filter.NewBloomFilter(16)
 	blockDb, err := openLevelDBWithOptions(blockStorage, &blockDbOptions)
 	if err != nil {
 		return nil, err
```

Comment:
@strib when I was referring to the metadata DB, I meant the one right underneath here. That should give the biggest wins.

Reply:
@jzila I know.  That one has it too.  It used the default `openLevelDB()` function which populates it.  I only added it explicitly to the DBs that use `openLevelDBWithOptions`.

That said, I couldn't measure a win with that DB.

Reply:
Ah I missed that the default options have it now too, thanks for the clarification.